bitbots_utils/launch/tools.launch

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0"?>
 <launch>
     <arg name="monitoring" default="true" description="true: start system monitor and udp bridge" />
+    <arg name="recording" default="true" description="true: start the ros bag recording" />
```

Comment:
In `robocup_teamplayer.launch`, this argument is called `record`. I would prefer to keep the name consistent across launch files.